src/app/shared/conditional-show/conditional-show.directive.ts

```diff
@@ -1,13 +1,14 @@
 import { AfterViewInit, Directive, ElementRef, Input, OnDestroy } from '@angular/core';
```

Comment:
@tzarsmango haven't checked you PR yet. But I just noticed that we use the following method in this file:

 private getFormFieldsValuesIncludingDisabled() {
    return this.formGroup.getRawValue();
  }

I don't remember exactly the reason I decided to include also disabled fields when initially developed show condition, but, have you tried changing this method to use .value property rather than .getRawValue() in order to exclude disabled fields? Just worth mentioning, it might not work